www/js/lib/zimDirEntry.js

```diff
@@ -107,6 +107,15 @@ define([], function() {
         return new DirEntry(zimfile, data);
     };
 
+    /**
+     * Defines a getter function that returns the URL if the title is empty
```

Comment:
Here, I would not say it is "getter function" (might be confusing).
And I would refer to the spec to explain why we implement this fallback

Reply:
OK, will do.